inst/bs3compat/_navbar_compat.scss

```diff
@@ -101,15 +101,6 @@ ul.nav.navbar-nav {
   }
 }
 
-$enable-dark-mode: false !default;
-@if $enable-dark-mode {
-  @include color-mode(dark) {
-    .navbar.navbar-default {
-      background-color: var(--bslib-navbar-default-bg, var(--#{$prefix}dark)) !important;
-    }
-  }
-}
```

Comment:
Let's make sure to test with `version = 4` as well (and if it's needed for old version(s), we can bring in these rules conditionally based on `$bootstrap-version`

Reply:
You added this in 

* https://github.com/rstudio/bslib/pull/764

... and BS4 doesn't have an `$enable-dark-mode` flag.